src/app/photos/photos.component.html

```diff
@@ -0,0 +1,12 @@
+<li>
```

Comment:
For consistency, please change your IDE's default indentation to use 2 spaces instead of 4.